### ISSUE TITLE
Error ObjectUnsubscribedError when going back to the page from routerLink

### ISSUE THREAD

**Issue by zphingphong** [arkon/ng-sidebar#92]
The issue is similar to https://github.com/arkon/ng-sidebar/issues/73.

```
ERROR Error: Uncaught (in promise): ObjectUnsubscribedError: object unsubscribed
ObjectUnsubscribedError: object unsubscribed
    at new ObjectUnsubscribedError (ObjectUnsubscribedError.js:19)
    at EventEmitter.Subject._trySubscribe (Subject.js:94)
    at EventEmitter.Observable.subscribe (Observable.js:45)
    at EventEmitter.subscribe (core.es5.js:3873)
    at SidebarContainer._subscribe (sidebar-container.component.js:133)
    at SafeSubscriber._next (sidebar-container.component.js:26)
    at SafeSubscriber.__tryOrSetError (Subscriber.js:247)
    at SafeSubscriber.next (Subscriber.js:187)
    at Subscriber._next (Subscriber.js:125)
    at Subscriber.next (Subscriber.js:89)
    at new ObjectUnsubscribedError (ObjectUnsubscribedError.js:19)
    at EventEmitter.Subject._trySubscribe (Subject.js:94)
    at EventEmitter.Observable.subscribe (Observable.js:45)
    at EventEmitter.subscribe (core.es5.js:3873)
    at SidebarContainer._subscribe (sidebar-container.component.js:133)
    at SafeSubscriber._next (sidebar-container.component.js:26)
    at SafeSubscriber.__tryOrSetError (Subscriber.js:247)
    at SafeSubscriber.next (Subscriber.js:187)
    at Subscriber._next (Subscriber.js:125)
    at Subscriber.next (Subscriber.js:89)
    at resolvePromise (zone.js:769)
    at resolvePromise (zone.js:740)
    at zone.js:817
    at ZoneDelegate.invokeTask (zone.js:424)
    at Object.onInvokeTask (core.es5.js:4140)
    at ZoneDelegate.invokeTask (zone.js:423)
    at Zone.runTask (zone.js:191)
    at drainMicroTaskQueue (zone.js:584)
    at ZoneTask.invoke (zone.js:490)
    at timer (zone.js:1512)
```

I found that this happens when you go back to previous page, which has the sidebar, from a page you are on.

Steps to reproduce
1. Sidebar is on page A
2. Navigate from page A to B but clicking on a `routerLink`
3. Click browser's back button

page A code

```
<ng-sidebar-container>
  <ng-sidebar #sidebar [(opened)]="sidebarOpened" [showBackdrop]="true" [closeOnClickBackdrop]="true">
    <p>Sidebar contents</p>
    <button (click)="sidebar.close()">Close sidebar</button>
  </ng-sidebar>
  <button (click)="toggleSidebar()">Toggle sidebar</button>
</ng-sidebar-container>

<a [routerLink]="['/pageB']">
  Go To Page B
</a>
```

page B has nothing but a text `TEST`.

What I found is that `SidebarContainer._subscribe()` is called twice, when you're back on the page. `onRegister` in `SidebarContainer.ngAfterContentInit` is called twice.

**Comment by arkon** [comment]
Fixed in v5.1.0.

**Comment by zphingphong** [comment]
Thanks @arkon. That's quick!
